### ISSUE TITLE
description:"Error received from peer"

### ISSUE THREAD

**Issue by weizhenzhao** [tensorflow/serving#1149]
Hi Dear

I install the tensorflow serving from the docker image
and run it in the docker like the following
![image](https://user-images.githubusercontent.com/13706689/47069321-303f2b00-d221-11e8-9b49-fa0f1ff05cea.png)
![image](https://user-images.githubusercontent.com/13706689/47069753-4d282e00-d222-11e8-994f-5e67b961e323.png)

And then I call it with the client script ,
But I got an error as following:
grpc._channel._Rendezvous: <_Rendezvous of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "Trying to connect an http1.x server"
	debug_error_string = "{"created":"@1539760804.602000000","description":"Error received from peer","file":"src/core/lib/surface/call.cc","file_line":1095,"grpc_message":"Trying to connect an http1.x server","grpc_status":14}"
>
"

And my model's signature define is as below:
signature_def {
    key: "predict_classes"
    value {
      inputs {
        key: "inputString"
        value {
          name: "input_x:0"
          dtype: DT_INT32
          tensor_shape {
            dim {
              size: -1
            }
            dim {
              size: 40
            }
          }
        }
      }
      outputs {
        key: "predictClass"
        value {
          name: "output/predictions:0"
          dtype: DT_INT64
          tensor_shape {
            dim {
              size: -1
            }
          }
        }
      }
      method_name: "tensorflow/serving/predict"
    }
  }
  signature_def {
    key: "serving_default"
    value {
      inputs {
        key: "inputs"
        value {
          name: "input_x:0"
          dtype: DT_INT32
          tensor_shape {
            dim {
              size: -1
            }
            dim {
              size: 40
            }
          }
        }
      }
      outputs {
        key: "classes"
        value {
          name: "index_to_string_Lookup:0"
          dtype: DT_STRING
          tensor_shape {
            dim {
              size: 1
            }
          }
        }
      }
      outputs {
        key: "scores"
        value {
          name: "output/scores:0"
          dtype: DT_FLOAT
          tensor_shape {
            dim {
              size: -1
            }
            dim {
              size: 2
            }
          }
        }
      }
      method_name: "tensorflow/serving/classify"
    }
  }
}

my client code is as below
`
def main(_):
  host, port = FLAGS.server.split(':')
  channel = implementations.insecure_channel(host, int(port))
  stub = prediction_service_pb2.beta_create_PredictionService_stub(channel)
  testStr =['about 25 minutes of decent material']
  request = predict_pb2.PredictRequest()
  request.model_spec.name = 'text_classified_model'
  request.model_spec.signature_name = 'predict_classes'
  
  x_text, y = data_helpers.load_data_and_labels(FLAGS.positive_data_file, FLAGS.negative_data_file)
  max_document_length = max([len(x.split(" ")) for x in x_text])

  vocab_processor = learn.preprocessing.VocabularyProcessor(max_document_length)
  vocab_processor.fit(x_text)
  x = np.array(list(vocab_processor.fit_transform(testStr)))
  
  request.inputs['inputString'].CopyFrom(
  tf.contrib.util.make_tensor_proto(x, shape=[1,40]))
  result = stub.Predict(request, 10.0)  # 10 secs timeout
  print(result)

if __name__ == '__main__':
  tf.app.run()

`

Can anyone give me some advice?
Thank you very much 
Weizhen

**Comment by gautamvasudevan** [comment]
Please go to Stack Overflow for help and support:

https://stackoverflow.com/questions/tagged/tensorflow-serving

If you open a GitHub issue, it must be a bug, a feature request, or a
significant problem with documentation (for small docs fixes please send
a PR instead).

Thanks!
